indexers/src/main/java/org/mousephenotype/cda/indexers/ObservationIndexer.java

```diff
@@ -288,13 +292,10 @@ public long populateObservationSolrCore(RunStatus runStatus) throws SQLException
 						}
 
 						OntologyDAO ontoService = null;
-						OntologyTermHelper b = null;
 						if (anatomyTermId.startsWith("MA:")) {
 							ontoService = maOntologyService;
-
 						} else if (anatomyTermId.startsWith("EMAP:")) {
 							ontoService = maOntologyService;
```

Comment:
This looks like an existing bug.  Should this set ontoService to emapOntologyService ?